src/main/java/seedu/address/logic/position/EditPositionCommand.java

```diff
@@ -100,6 +101,13 @@ public CommandResult execute(Model model) throws CommandException {
             }
         }
 
+        for (Interview interview : lastShownInterviewList) {
+            Position interviewPosition = interview.getPosition();
+            if (interviewPosition.isSamePosition(positionToEdit)) {
+                interview.setPosition(editedPosition);
+            }
+        }
+
```

Comment:
Currently edit_p does not have immediate GUI update, perhaps can add 
"model.updateFilteredInterviewList(PREDICATE_SHOW_ALL_INTERVIEWS);"

Reply:
Oh wait it is handled by your other PR never mind just ignore this